tools/xxhash/Makefile

```diff
@@ -0,0 +1,24 @@
+# SPDX-License-Identifier: GPL-2.0-only
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=xxHash
+PKG_VERSION:=0.8.3
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/Cyan4973/xxHash/tar.gz/v$(PKG_VERSION)?
```

Comment:
any chance we can use the git clone way or other alternative method than codeload?

Reply:
```
PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
PKG_SOURCE_URL_FILE:=v$(PKG_VERSION).tar.gz
PKG_SOURCE_URL:=https://github.com/Cyan4973/xxHash/archive/refs/tags
```
Is this acceptable?